src/components/Header/styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+const HeaderImage = styled.img`
+  margin-right: 100px;
```

Comment:
Consider using `em` and `rem` values to be very flexible with font-size, padding, margin etc, this helps a lot when doing the responsive UI designs.